go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/google/uuid v1.2.0
 	github.com/gruntwork-io/terratest v0.37.7
-	github.com/k8ssandra/cass-operator v1.9.0
+	github.com/k8ssandra/cass-operator v1.8.0-rc.2.0.20220114225934-6a78a8237173
```

Comment:
That's what go mod update added.

Reply:
Doesn't make it correct even if it occasionally fetches wrong version. I wonder why it would pull something as old, is something in the reaper/medusa-operator merges using a feature that caused this?

Reply:
That version is actually correct. I don't understand the `v1.8.0-rc.2.0` part. The next part though is date followed by the commit hash of the latest commit in master.